src/iter.rs

```diff
@@ -25,14 +23,14 @@ impl<'a> Bytes<'a> {
     }
 
     #[inline]
-    pub unsafe fn bump(&mut self) {
```

Comment:
This function can violate a struct invariant, so unless the `debug_assert` is changed to `assert` it must be marked as `unsafe` or it would be unsound.